ssg/build_yaml.py

```diff
@@ -612,6 +613,11 @@ def validate_references(self, yaml_file):
                 raise ValueError("References must not be empty: %s in file %s"
                                  % (ref_type, yaml_file))
 
+    def normalize_references(self):
+        for ref_type, ref_val in self.references.items():
+            csv_elements = [ref.strip() for ref in ref_val.split(",")]
+            self.references[ref_type] = ",".join(csv_elements)
```

Comment:
Should we issue an error and fail the build instead of fixing it silently?

Reply:
I have also thought of that. I am slightly in favor of being more strict, but I have anticipated that the community may be against errors like that.

Reply:
@matejak I am completely in favor of being more strict in all YAML keys that contain comma-separated lists with a failure. While not the scope of this PR, it would also be nice to fail on trailing whitespace.

Reply:
These errors will be easy to find and fix.